drivers/hmis/app/graphql/types/hmis_schema/validation_error.rb

```diff
@@ -10,41 +10,69 @@ module Types
   class HmisSchema::ValidationError < Types::BaseObject
     field :id, String, null: true
     field :attribute, String, null: true
+    field :readable_attribute, String, null: true
     field :message, String, null: false
-    field :full_message, String, null: true
-    field :type, String, null: false
+    field :full_message, String, null: false
+    field :type, HmisSchema::Enums::ValidationType, null: false
+    field :severity, HmisSchema::Enums::ValidationSeverity, null: false
     field :options, JsonObject, null: true
 
     def attribute
-      return object.attribute.to_s.underscore.camelize(:lower) if object.respond_to?(:attribute)
+      object.attribute.to_s.underscore.camelize(:lower) if object.respond_to?(:attribute)
```

Comment:
this might be a different camelize, but it looks like it should be
```suggestion
      object.attribute.to_s.underscore.camelize(true) if object.respond_to?(:attribute)
```
https://apidock.com/rails/v6.1.3.1/String/camelize

Reply:
Rails 7 seems to support `:lower`
https://api.rubyonrails.org/classes/String.html#method-i-camelize

Reply:
Huh, are we on Rails 7 or 6? I thought we were on six. But passing `true` here gives me `ArgumentError (Invalid option, use either :upper or :lower.`

Reply:
yeah, were on 6.1, maybe `camelize` is coming from somewhere else.  If it's working as expected, let's not touch it.